### PR TITLE
fix-json-xml

### PR DESCRIPTION
**Pull request by michmuel**
closes https://github.com/openoereb/pyramid_oereb/issues/1382

### CODE REVIEW

**Comment by vvmruder** [comment]
@michmuel Can you check? My first quick check confirms working code now. Please have a look on edge cases.

Copy is now available at the document record itself. I think this is more reliable and transparent.

**Comment by michmuel** [comment]
@vvmruder 
- function "get_law_status_by_code_main_extract" in config.py deleted (or is it necessary for anything?)
- theme_documents = [] in config.py within loop for themes (every theme has another set of documents)
- document_types.lookup removed in pyramid_oereb.yml.mako (or is it necessary for anything?)
- fix code-style issues (lint)

File: pyramid_oereb/lib/config.py [hunk 1]
```diff
@@ -129,14 +129,15 @@ def init_offices():
 
     @staticmethod
     def assemble_relation_themes_documents():
-        theme_documents = []
         for theme in Config.themes:
+            theme_documents = []
```
Comment (vvmruder) [inline pyramid_oereb/lib/config.py:133]: nice catch

**Review by vvmruder — APPROVED** [review]

**Comment by vvmruder** [comment]
I also removed some unused methods. Now it should be ready :) Thank you @michmuel for checking and improving

**Comment by michmuel** [comment]
```
============================================================================================== short test summary info ==============================================================================================
ERROR tests/init_db.py - sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) null value in column "id" of relation "real_estate_type" violates not-null constraint
ERROR tests/renderer/test_xml.py - pyramid.exceptions.ConfigurationError: Logo for code: ch.plr not found in the application configuration
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Pull request merged**